### PR TITLE
Added support for player-type

### PR DESCRIPTION
**Pull request by martinvw**
Hello,

I needed this code to start/stop my HEOS so I added the support for the player-item.

Feedback is welcome!

### CODE REVIEW

File: openhab/types.py [hunk 1]
```diff
@@ -256,3 +256,24 @@ def validate(cls, value: str) -> None:
...
+    Args:
+      value (str): The value to validate.
+
+    Raises:
+      ValueError: Raises ValueError if an invalid value has been specified.
+    """
+    super().validate(value)
+
+    if value not in ['PLAY', 'PAUSE', 'NEXT', 'PREVIOUS', 'REWIND', 'FASTFORWARD']:
+      raise ValueError()
```
Comment (sim0nx) [inline openhab/types.py]: Please add a newline at the end of the file

**Review by sim0nx — CHANGES_REQUESTED** [review]

**Comment by sim0nx** [comment]
Hi
Beside the tiny comment, looks good to me, thanks

File: openhab/types.py (hunk 1 above)
Comment (martinvw) [reply openhab/types.py]: Updated!

**Comment by martinvw** [comment]
It seems one of the tests broke, I see no reason what could cause that. If I did it I might need a hint 😊

**Review by sim0nx — APPROVED** [review]

**Comment by sim0nx** [comment]
> It seems one of the tests broke, I see no reason what could cause that. If I did it I might need a hint

Not sure why but sometimes travis checks break for no reason. 3.7 ran without any issues (and there are none), so just ignore it for now :-)

**Pull request merged**